test/java/io/envoyproxy/envoymobile/engine/EnvoyConfigurationTest.kt

```diff
@@ -45,6 +45,7 @@ class EnvoyConfigurationTest {
     assertThat(resolvedTemplate).contains("&dns_fail_base_interval 345s")
     assertThat(resolvedTemplate).contains("&dns_fail_max_interval 456s")
     assertThat(resolvedTemplate).contains("&dns_query_timeout 321s")
+    assertThat(resolvedTemplate).contains("&dns_lookup_family V4_PREFERRED")
```

Comment:
Can you add Kotlin & Swift tests validating enabling happy eyeballs?

Reply:
Added some extra tests.

Reply:
@jpsim did you mean at the config level or at the functionality level?

Reply:
The extra unit tests I added are I think the extent to which this is reasonably testable at this level. The actual Happy Eyeballs implementation itself will of course have its own tests in upstream Envoy.

Reply:
The config level is fine here.